### ISSUE TITLE
undefined is not a function

### ISSUE THREAD

**Issue by shammar13** [fitnr/taft#3]
Hi guys

I think there is an error here with your `tostring()` method. I think it should be `toString()`:
https://github.com/fitnr/taft/blob/master/bin/taft.js#L85

`tostring()` is generating the following error:

``` js
building: path/to/template/test.hbs
c:\path\to\node_modules\taft\bin\taft.js:85
            save(outfile, build.tostring());
TypeError: undefined is not a function
    at c:\path\to\node_modules\taft\bin\taft.js:85:33
    .
    .
    .
```

Thanks!

**Comment by fitnr** [comment]
Shoot, just introduced that yesterday. I'll push a fix right away

**Comment by fitnr** [comment]
Fixed by 290b8f3. Thanks for bringing it up!
